geode-core/src/main/java/org/apache/geode/cache/client/internal/EndpointManagerImpl.java

```diff
@@ -181,6 +182,11 @@ public void removeListener(EndpointManager.EndpointListener listener) {
     this.listener.removeListener(listener);
   }
 
+  @TestOnly
```

Comment:
Do we have guidelines on when to use `@TestOnly` versus `@VisibleForTesting`?

Reply:
No, but I don't want these API to be used at all outside of teset, not just to be private.

Reply:
Removed unit tests.